### ISSUE TITLE
Add button to show ActionSheet

### ISSUE THREAD

**Issue by antonioloiaconodev** [nikini/ionic-gallery-modal#11]
How to add button to show actionSheet into modal for more function!?

**Comment by nikini** [comment]
It's not possible right now. Will think about it for future versions

**Comment by jayserdny** [comment]
@nikini I would like to implement this enhancement 💯 @kadox07 Can you tell me more about what you need?

**Comment by nikini** [comment]
I think that what @kadox07 needs is that he can have interaction in the gallery and maybe when you tap on an image, that it can show an actionsheet. But, I wouldn't make it specifically that, but more that allow interaction on images. Does that make sense, @jayserdny ?

**Comment by jayserdny** [comment]
@nikini Maybe a functionality like Facebook can be added where users are able to download or share the current photo in the gallery.

**Comment by nikini** [comment]
Yeah, something like that. But I wouldn't say that it should be integrated into the plugin, but more of a capability that developers should have to build themselves on top of the plugin. Don't you agree, @jayserdny ?

**Comment by jayserdny** [comment]
@nikini I agrre with you. Since some developers have different ideas, I believe the enhancement should allow the developer to implement a header or footer to the modal page with their own option. What do you think? 

By the way, I was playing around with your plugin and I added option to show pagination in the slidesm

**Comment by Flink91** [comment]
I want to add a download button as well, any news on that?
